chrisparsonsdev-London.md

```diff
@@ -0,0 +1,10 @@
+# London, UK
+
+### Restaurants
+
+### Local Attractions
+
+- London Eye
```

Comment:
@ChrisParsonsDev I have never been there, I am from California. Would like to visit London Eye some day